README.md

```diff
@@ -1,54 +1,49 @@
 # Fedora Cosmic Atomic
-### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment Included!
+### Fedora Silverblue-based Atomic Desktop with the Pre-Alpha Cosmic Desktop Environment included!
 
-> NOTE: These images are not associated with System76! If you have issues, please understand they might be COSMIC related, OR they might be related to this image.
+> NOTES:
+> These images are not associated with System76! If you have issues, please understand they might be COSMIC related, OR they might be related to this image.
+> The COSMIC Desktop Environment is still PRE ALPHA. Do not daily drive this image on your main workstation unless you know what you're doing.
 
 Like Fedora? Want to try the latest from the work in progress Cosmic Desktop Environment? Want to help find bugs and/or contribute to Cosmic development, but don't want to work in a VM or install Pop!_OS? None of the above things but something else??!?
 
 Go ahead and try one of the ostree images I've created here!
 
 ### Quick Installation
 
-Install an rpm-ostree based desktop, like [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/) (aka atomic desktops)
+Install a Fedora Atomic Desktop, best [Fedora Silverblue](https://fedoraproject.org/atomic-desktops/silverblue/).
```

Comment:
best Fedora Silverblue? 

This wording should be fixed back to 'like'.

Reply:
this was given the fact that it was previously a Fedora silverblue based image. will revert it back if you think that makes sense.